### PR TITLE
removing PlayFab SDK from Unity GSDK

### PR DESCRIPTION
**Pull request by dgkanatsios**
This PR removes PlayFab SDK from Unity GSDK. It keeps only two files relevant to JSON (de)serialization, residing in a different folder/namespace so there will be no conflict if the Unity developer installs PlayFab SDK.

### CODE REVIEW

File: UnityGsdk/Assets/PlayFabSdk/MultiplayerAgent/Helpers/ISerializer.cs [hunk 1]
```diff
@@ -78,6 +78,31 @@ protected override bool TrySerializeKnownTypes(object input, out object output)
                 }
                 return base.TrySerializeKnownTypes(input, out output);
             }
+            
+            public static readonly string[] _defaultDateTimeFormats = new string[]{ // All parseable ISO 8601 formats for DateTime.[Try]ParseExact - Lets us deserialize any legacy timestamps in one of these formats
```
Comment (yzhao174) [inline UnityGsdk/Assets/PlayFabSdk/MultiplayerAgent/Helpers/ISerializer.cs:82]: >_defaultDateTimeFormats [](start = 44, length = 23)

Very minor: Our naming convention for "static readonly" is "DefaultDateTimeFormats"

File: UnityGsdk/Assets/PlayFabSdk/MultiplayerAgent/Helpers/ISerializer.cs [hunk 2]
```diff
@@ -1,18 +1,18 @@
...
-
-namespace PlayFab.Json
+﻿namespace PlayFab.MultiplayerAgent.Helpers
 {
-    public class SimpleJsonInstance : ISerializerPlugin
+    using System;
+    using System.Globalization;
+    using System.Reflection;
+
+    public class SimpleJsonInstance
```
Comment (yzhao174) [inline UnityGsdk/Assets/PlayFabSdk/MultiplayerAgent/Helpers/ISerializer.cs:7]: >SimpleJsonInstance [](start = 17, length = 18)

Nit: The name of this class doesn't quit match the file name.

Comment (dgkanatsios) [reply UnityGsdk/Assets/PlayFabSdk/MultiplayerAgent/Helpers/ISerializer.cs:7]: nice catch, fixing!

File: UnityGsdk/Assets/PlayFabSdk/MultiplayerAgent/Helpers/ISerializer.cs (hunk 1 above)
Comment (dgkanatsios) [reply UnityGsdk/Assets/PlayFabSdk/MultiplayerAgent/Helpers/ISerializer.cs:82]: fixing

**Pull request merged**